### PR TITLE
DOCSP-26498 - Standardize MongoDB Server Naming

### PR DESCRIPTION
**Pull request by mongoKart**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - https://jira.mongodb.org/browse/DOCSP-26498
Staging - https://docs-mongodbcom-staging.corp.mongodb.com/java/docsworker-xlarge/docsp-26498-mongodb-caps/

## Self-Review Checklist

- [X] Is this free of any warnings or errors in the RST?
- [X] Did you run a spell-check?
- [ ] Did you run a grammar-check?
- [ ] Are all the links working?

### CODE REVIEW

File: source/fundamentals/crud/write-operations/change-a-document.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/crud/write-operations/change-a-document.txt:132]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

```suggestion
    collection. For more information about unique index constraints, see 
    :manual:`unique indexes </core/index-unique/>` in the {+mdb-server+} manual. 
```

File: source/fundamentals/crud/write-operations/change-a-document.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/crud/write-operations/change-a-document.txt:209]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:168]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:203]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:309]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:364]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:396]: [suggestion]

Per our [Style Guide](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear), we should put the reason why we have the link before we say "see X".

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [inline source/fundamentals/indexes.txt:421]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/bulkWrite.txt
Comment (DBirtolo-mdb) [inline source/usage-examples/bulkWrite.txt:40]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/command.txt
Comment (DBirtolo-mdb) [inline source/usage-examples/command.txt:9]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/count.txt
Comment (DBirtolo-mdb) [inline source/usage-examples/count.txt:46]: [question]

Is there a reason this is not {+mdb-server+}?

File: source/usage-examples/count.txt
Comment (DBirtolo-mdb) [inline source/usage-examples/count.txt:49]: [question]

Is there a reason this is not {+mdb-server+}?

**Review by DBirtolo-mdb — CHANGES_REQUESTED** [review]
Nice job! Mainly two things (but there is at least one other one-off comment in there):

* Using the placeholder constant text
* Recommended link text updates

File: source/fundamentals/crud/write-operations/change-a-document.txt
Comment (mongoKart) [reply source/fundamentals/crud/write-operations/change-a-document.txt:132]: Sure. Wasn't looking at things like that for this ticket, but happy to change.

File: source/fundamentals/indexes.txt
Comment (mongoKart) [reply source/fundamentals/indexes.txt:309]: This looks okay to me...?

File: source/fundamentals/indexes.txt
Comment (DBirtolo-mdb) [reply source/fundamentals/indexes.txt:88]: Missed this one, but otherwise all good!

**Review by DBirtolo-mdb — APPROVED** [review]
LGTM

There was one instance where you didn't replace "MongoDB Server" with the constant (in source/fundamentals/indexes.txt), but that wasn't a critical change, so didn't want to hold up approval.

**Pull request merged**
